### PR TITLE
Updating README adjusting to GPU support 

### PR DESCRIPTION
**Pull request by Carmezim**
Updating current limitations and adding instructions to use GPU on Cmake build.

### CODE REVIEW

**Comment by thepok** [comment]
Maybe mention that a ^ has to be added to 
More? -DPYTHON_LIBRARIES=C:/Users/%USERNAME%/AppData/Local/Continuum/Anaconda3/libs/python35.lib
if you want to add more lines.

**Comment by Carmezim** [comment]
@thepok May bad. I am doing it now. Thank you!

**Comment by thepok** [comment]
The last steps in the Readme miss, it took me a long time to figure it out. I did not know what to do with the genereatet PIP file. Maybe add a line describing that you have to do "PIP install genereted_file.whl" or what ever is the real name :)

**Comment by drpngx** [comment]
@Carmezim do you want to update the README?

**Comment by Carmezim** [comment]
@drpngx I just wanted to make those changes on the Cmake readme to avoid confusing folks regarding GPU support as it is already supported and the readme states otherwise.

**Comment by drpngx** [comment]
Jenkins, test this please.

**Comment by drpngx** [comment]
What happens after CMake 3.6?

**Comment by Carmezim** [comment]
@drpngx when building with v.37 it generates an error referring to a stray character " after "MSVC" in line 81 of `tf_core_gpu_kernels_generated_adjust_contrast_op_gpu.cu.cc.obj.RelWithDebInfo.cmake`, earlier versions build successfully [#5071](https://github.com/tensorflow/tensorflow/pull/5071#issuecomment-255501356).

**Comment by drpngx** [comment]
Got it, @mrry and @guschmue might be interested in knowing that CMakeList is broken after 3.6.

**Comment by Carmezim** [comment]
@drpngx @guschmue knows as he figured it out with folks testing. Now it seems [#5127](https://github.com/tensorflow/tensorflow/pull/5127) came with something that is generating other errors though that even with v3.6 it is not building successfully.

**Comment by drpngx** [comment]
Thanks for identifying the CL!

**Comment by guschmue** [comment]
I looked at #5127 and think I have a simple fix for it. Just building a fix and if good I'll send a PR.

**Review by guschmue — APPROVED** [review]
cmake 3.7 does NVCC_OPTIONS correctly. We think it stubles over -D__VERSION__=\"MSVC\". Going to look if we can do something on this but for now its good to ask people to use 3.6 so they don't waste their time.

**Comment by zheng-xq** [comment]
@guschmue, could you look at the latest changes and see if you can LGTM? Thanks.

File: tensorflow/contrib/cmake/README.md
Comment (Carmezim) [inline tensorflow/contrib/cmake/README.md]: Adding it. Could you please just check if you think the instructions placement is appropriated on Step-by-Step-Build.

**Review by guschmue — COMMENTED** [review]
LGTM if you fix the PATH mentioned in my comment earlier.

**Comment by zheng-xq** [comment]
@guschmue, could you do one final pass before we merge this? Thanks.

File: tensorflow/contrib/cmake/README.md
Comment (Carmezim) [inline tensorflow/contrib/cmake/README.md]: Do you rather put the set instruction like:
`set PATH="%PATH%;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v8.0\bin;c:\local\cuda\bin`
?

**Review by guschmue — COMMENTED** [review]
LGTM

**Pull request merged**
